mitosheet/src/components/modals/ErrorModal.tsx

```diff
@@ -65,7 +65,7 @@ const ErrorModal = (
                     <TextButton
                         variant='dark'
                         width='medium'
-                        href={DISCORD_INVITE_LINK}
+                        href={SLACK_INVITE_LINK}
```

Comment:
I'd be down to randomize between the two of them, and see what people click more!

Reply:
For now, I think the most important thing is just engagement. Discord has almost 0 engagement, so let's see if people are on Slack more and will post more. This button also just says "Get immediate support" anyways.